### ISSUE TITLE
This project appears dead, should someone fork it?

### ISSUE THREAD

**Issue by grintor** [securitytxt/security-txt#215]
There has been no activity in this repo in months. The ietf draft linked on the homepage expired in November. The securitytxt.io domain linked in the readme (and configured as the deployment target for github pages) expired long ago and it available to purchase now.

Maybe there is activity happening somewhere, but I can't see it. Regardless, this is a worthwhile cause. I am willing to fork it and continue the work if the project is abandoned. If it is not abandoned, can someone chime in?

**Comment by nightwatchcyber** [comment]
Hi - this project is not dead. Our draft is not expired but went into final approval stage at the IETF in June of 2021 and got stuck there for a while, but it is finally unstuck and should become an RFC within the next few weeks. You can see the queue here:
https://www.rfc-editor.org/current_queue.php

Once the RFC is approved, we are planning on re-start work on multiple issues that have been on hold until now.

**Comment by EdOverflow** [comment]
The `securitytxt.io` domain merely redirected to the `security.txt` Internet Draft (ID). I decided to discontinue using it since the draft will be released as RFC 9116 soon which is more memorable and discoverable than the ID URI.
